src/serverModuleGen.py

```diff
@@ -7,20 +7,22 @@ def getModules(filename):
         modules = configfile.readlines()
         ret = []
         for module in modules:
-            module = module.split("#")[0].split("/")[-1].strip()
+            module = module.split("#")[0].strip()
             if module:
                 ret.append(module)
         return ret
 
 
-def generateServerIncludes(config_filename, reg_filename):
-    serverfile = open(reg_filename, "w")
-    serverfile.write("proc doRegister() {\n")
+def generateServerIncludes(config_filename, src_dir):
+    res = ""
     for mod in getModules(config_filename):
-        serverfile.write(f"  import {mod};\n  {mod}.registerMe();\n")
+        if mod[0] != '/':
+            res += f" {src_dir}/{mod}.chpl"
+        else:
+            res += f"{mod}.chpl"
 
-    serverfile.write("}\n")
+    print(res)
```

Comment:
I noticed the `if` starts with a space and the `else` doesn't, just wanted to check that was intentional and not overlooked

okay also i couldn't help but try and code golf this
```python
print("".join([f" {src_dir}/{mod}.chpl" if mod[0] != "/" else f"{mod}.chpl" for mod in getModules(config_filename)]))
```

Reply:
Ooh, good catch, that was overlooked by me, it should have a space.

While I am continually impressed by the code gold syntax, I have to say I am not the biggest fan. In my opinion, the code just becomes a little bit harder to read/maintain (it is cool though).

Reply:
That's fair! I certainly can't argue it's more readable lol. But it is more fun to write!

I know dict/list comprehensions can have slightly better performance but that hardly matters here